docs/txn-advanced-features.md

```diff
@@ -0,0 +1,45 @@
+---
+id: txn-advanced-features
+title: Advanced features
+sidebar_label: "Advanced features"
+---
+
+You can use the following advanced features with transactions in Pulsar.
+
+## Ack batch messages
+
+If you want to acknowledge batch messages with transactions, set `acknowledgmentAtBatchIndexLevelEnabled` to `true` in the [`broker.conf`](https://github.com/apache/pulsar/blob/master/conf/broker.conf) or [`standalone.conf`](https://github.com/apache/pulsar/blob/master/conf/standalone.conf) file.
+
+
+```conf
+acknowledgmentAtBatchIndexLevelEnabled=true
+```
+
+This example enables batch messages ack in transactions in the consumer builder.
+
+```java
+Consumer<byte[]> consumer = pulsarClient
+    .newConsumer()
+    .topic(transferTopic)
+    .subscriptionName("transaction-sub")
+    .subscriptionInitialPosition(SubscriptionInitialPosition.Earliest)
+    .subscriptionType(SubscriptionType.Shared)
+    .enableBatchIndexAcknowledgment(true) // enable batch index acknowledgment
+    .subscribe();
+```
+
+## Enable authentication
```

Comment:
I think Enable OAuth2 authentication is much appropriate as there are multiple authentication methods and here we focus on OAuth2 auth

Reply:
@congbobo184 double check: can users enable OAuth2 auth only or all methods?
<img width="277" alt="image" src="https://user-images.githubusercontent.com/50226895/229676347-33e15be8-17b5-451e-8bc3-a95c7d0e3584.png">

Reply:
I think it is better to directly configure the link of auth, like: https://pulsar.apache.org/docs/2.11.x/Authentication, but now don't have the link of this

Reply:
@congbobo184 Oh I see, you mean users can enable all Authentication methods.

So I plan to set it as https://pulsar.apache.org/docs/next/security-overview/#authentication, which directs users to all methods as well. WDYT?

<img width="1431" alt="image" src="https://user-images.githubusercontent.com/50226895/230249813-98a92636-2c9c-4544-bb76-e07a7aa071f3.png">

Reply:
yes, directs users to all methods as well. can use https://pulsar.apache.org/docs/next/security-overview/#authentication